app/code/core/Mage/Core/Model/App.php

```diff
@@ -36,12 +36,6 @@ class Mage_Core_Model_App
 
     public const DISTRO_LOCALE_CODE = 'en_US';
 
-    /**
-     * Cache tag for all cache data exclude config cache
-     *
-     */
-    public const CACHE_TAG = 'MAGE';
-
```

Comment:
I guess PHPStan is right by removing this line we get in many places "Access to undefined constant Mage_Core_Model_App::CACHE_TAG.". I am seeing this error when opening the PR in browser.

Reply:
yes, the 3 methods mentioned in the PR description that still have to be fixed (first we have to decide how to) ;-)